content/beginner/300_windows/calico_windows/install_calico_node.md

```diff
@@ -58,4 +58,4 @@ Running  kubelet         kubelet service
 Running  kube-proxy      kube-proxy service
 {{< /output >}}
 
-Now we have Calcio fully installed. Close the EC2 Connect window and return to Cloud9.
\ No newline at end of file
+Now we have Calcio fully installed. Close the EC2 Connect window and return to Cloud9.
```

Comment:
Thanks for this, can you also rename calcio to calico ?

Reply:
Done.